.ci_support/linux_python2.7.yaml

```diff
@@ -1,6 +1,14 @@
+c_compiler:
+- toolchain_c
+m2w64_c_compiler:
+- m2w64-toolchain
```

Comment:
Hmm...this is a bit surprising.

cc @isuruf

Reply:
Partially addressing in PR ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/80 ).